### DISCUSSION TITLE
keyball39_特定のキーが反応しない

### DISCUSSION THREAD

**Discussion by xiong2da4** [Yowkees/keyball#723] [Q&A]
keyball39を組み立て、6つ反応しないキーがありました。
キーソケットとダイオードのはんだ付けを再実行したところ、反応しないキーは4つに減りました。
ダイオードを変えてみたり、向きも確認したのですが、一向に改善しません。
何か解決策はありますでしょうか。
添付画像の赤枠で囲っている部分が、修正しようとしたキーソケットとダイオードのセットです。

![IMG_0213](https://github.com/user-attachments/assets/9236aceb-5a0a-4d02-b555-988a2f6b7bc6)
![IMG_0214](https://github.com/user-attachments/assets/cf54729d-dfe0-44b7-ae77-4305359a8944)

**Comment by xiong2da4** [comment]
キーソケットの端子と基盤の間に、肉眼で木確認できる隙間はありません、
キーソケットのはんだの修正は、はんだ吸い取り線でできる限りはんだを吸い取って、改めてはんだ付けをしています。
(残存しているはんだがありキースイッチを取り外すまでには至っていない)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

反応しないキーが１列全部ではなく、バラバラに点在していますので、キーソケットもしくはダイオードの半田付け不良の可能性が濃厚です。対応するダイオードの位置は赤枠で示していただいたもので合っています。

一見するとテンプラしていないように見えても、テンプラしている場合がありますので、
キーソケットの両端子をピンセット等でショートしてみて、反応がない場合はキーソケットの半田付けを見直してください。
長めに半田ゴテで熱して、端子と基板の間に溶けた半田が見えるかご確認いただければ幸いです。

以上、お手数ですがご対応お願いいたします。
